src/styles/blocks/main.scss

```diff
@@ -0,0 +1,7 @@
+.main {
+  display: grid;
+  grid-template: repeat(2 , 1fr) / repeat(4 , $card-width);
```

Comment:
please try to use `@media` when using grid: [readme](https://github.com/Sancz0pansa/layout_catalog/blob/develop/grid.md)